### PR TITLE
gsudo: Add version 0.4.1

### PR DESCRIPTION
**Pull request by gerardog**
Added gsudo console app: A sudo for windows with similar user experience as `*nix sudo` and credentials cache (avoid repeated UAC popups)

### CODE REVIEW

**Comment by h404bi** [comment]
```json
{
    "version": "0.4.1",
    "description": "A Sudo for Windows - run elevated without spanning a new Console Host Window",
    "homepage": "https://github.com/gerardog/gsudo",
    "url": "https://github.com/gerardog/gsudo/releases/download/v0.4.1/gsudo.v0.4.1.zip",
    "hash": "5a7c2bdaa5e2c399c99bd396fea6e35a3267c62d901e7326959b17b1abaf4a3b",
    "bin": [
        "gsudo.exe",
        [
            "gsudo.exe",
            "sudo"
        ]
    ],
    "checkver": {
        "github": "https://github.com/gerardog/gsudo"
    },
    "autoupdate": {
        "url": "https://github.com/gerardog/gsudo/releases/download/v$version/gsudo.v$version.zip"
    }
}
```

**Comment by linsui** [comment]
```
    "checkver": "github",
```

**Comment by gerardog** [comment]
> P.S. This can be moved to main bucket I guess.

I wish, really. I am the author. At this point it does not fits this main-bucket-criteria:
* a reasonably well-known and widely used developer tool

I started this project about a month ago.

File: bucket/gsudo.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.4.1",
+    "description": "A Sudo for Windows - run elevated without spanning a new Console Host Window",
```
Comment (Ash258) [inline bucket/gsudo.json]: No need for explanation

```suggestion
    "description": "A Sudo for Windows",
```

File: bucket/gsudo.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "homepage": "https://github.com/gerardog/gsudo",
```
[the remark this answers is not in the document]
Comment (h404bi) [reply bucket/gsudo.json]: the license is MIT

**Pull request closed without merging**